src/app/components/Header/Header.js

```diff
@@ -1,36 +1,57 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
 
 import NetworkSwitcher from '../NetworkSwitcher'
 import MainNav from '../MainNav'
 
-import './Header.css'
+import home from '../../../img/home.svg'
```

Comment:
What's the standard for naming SVGs in the React world? `home` seems a little unclear.

Reply:
I agree. Perhaps we can use homeSVG convention for icons?